crates/bevy_math/src/curve/mod.rs

```diff
@@ -968,6 +968,73 @@ where
     }
 }
 
+/// Convert a `start` and `end` point of some type into a [`Curve`], sampled by interpolating
+/// between the two boundary values and under the application of an easing function.
+pub fn easing_curve<T: Clone>(
+    domain: Interval,
```

Comment:
There is probably some discussion to be had here, but it seems like it might be preferable to just use the unit interval `[0, 1]` as the domain here and then let people use `reparametrize` if they want something else.

Reply:
Makes total sense. This will also make the whole thing much less complex in general (no divisions) which I favor!